mne/cuda.py

```diff
@@ -63,13 +63,14 @@ def init_cuda(ignore_config=False, verbose=None):
     # Triage possible errors for informative messaging
     _cuda_capable = False
     try:
-        import cupy
+        import cupy  # noqa
     except ImportError:
         warn('module cupy not found, CUDA not enabled')
         return
     try:
         # Initialize CUDA
-        cupy.cuda.Device()
+        device_id = int(get_config('MNE_CUDA_DEVICE', '0'))
```

Comment:
I would keep this outside the `try` block. We really want the `try` to just be about failure to init, while this could be a failure of the `get_config` value being set. `int(get_config(...))` where `get_config` gives a bad value will probably have an informative enough error on its own.